templates/webapi/branding/openSUSE/external_reporting.html.ep

```diff
@@ -14,6 +14,7 @@
     caasp => 'https://bugzilla.suse.com/enter_bug.cgi',
     openqa => 'https://progress.opensuse.org/projects/openqav3/issues/new',
     kubic => 'https://bugzilla.opensuse.org/enter_bug.cgi'
```

Comment:
you need to add a comma at the end of this line. And I consider it saver if you also add one at the next line as well. It's ok to end also the last list entry with a comma